src/lib.rs

```diff
@@ -117,7 +117,7 @@ where
 
     pub fn hard_reset<DELAY>(&mut self, delay: &mut DELAY) -> Result<(), ()>
     where
-        DELAY: DelayMs<u8>,
+        DELAY: DelayNs,
```

Comment:
What is the reason for changing to DelayNs? I think maybe not all devices support nanosecond resolution and we definitely don't need it here.

Reply:
Because it is the only trait which can be implemented in 1.0.0 if I haven't missed something.

Reply:
I see.